app/webpack/taxa/show/components/status_tab.jsx

```diff
@@ -141,6 +141,17 @@ const StatusTab = ( { statuses, listedTaxa, listedTaxaCount } ) => {
                       text={status.description}
                     />
                   ) }
+                  { status.taxon_id && status.taxon_name && status.taxon_id !== taxonId && (
+                      <div
```

Comment:
Please use two spaces for indentation. If you're using eslint it should alert you about that.